rpcd/playbooks/roles/rpc_setup_infrastructure/tasks/sshd.yml

```diff
@@ -0,0 +1,79 @@
+---
```

Comment:
The support role targets openstack service containers so we won't be able to do it there since the changes being made here need to be performed before anything else.